include/SFML/Graphics/Texture.hpp

```diff
@@ -80,10 +86,22 @@ class SFML_GRAPHICS_API Texture : GlResource
     Texture(const Texture& copy);
 
     ////////////////////////////////////////////////////////////
-    /// \brief Destructor
+    /// \brief Defaulted of copy operator
```

Comment:
What does "Defaulted of copy operator" mean?

Reply:
I've been staring at this trying to figure out what I was meaning. I have no idea. I'll get that fixed up really quick!